### PR TITLE
test(e2e): share via facebook

### PR DESCRIPTION
**Pull request by AntonOlkhovskyi**

### CODE REVIEW

File: .config/protractor.conf.js [hunk 1]
```diff
@@ -14,7 +14,10 @@ exports.config = {
     shardTestFiles: true,
     maxInstances: 4,
     chromeOptions: {
-      args: [ 'headless']
+      args: [ 'headless', '--window-size=1800x1080'],
```
Comment (ludmilanesvitiy) [inline .config/protractor.conf.js]: Are you sure that you need this specific size? maybe 1600*900 or 1920*1080 ?

File: .config/protractor.conf.js [hunk 2]
```diff
@@ -28,7 +31,7 @@ exports.config = {
   },
 
   specs: ['../test-e2e/app/Tests/**/*.e2e-spec.ts'],
-  exclude: ['../test-e2e/app/CMS/**/*.e2e-spec.ts', '../test-e2e/app/Tests/ClickEachLink.e2e-spec.ts'],
+  exclude: ['../test-e2e/app/CMS/**/*.e2e-spec.ts', '../test-e2e/app/Tests/ClickEachLink.e2e-spec.ts','../test-e2e/app/Tests/Facebook.e2e-spec.ts'],
```
Comment (ludmilanesvitiy) [inline .config/protractor.conf.js:34]: why do you excluding new test?

File: test-e2e/app/Tests/SocialNetworks/Facebook.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
+import { $, browser } from 'protractor';
```
Comment (ludmilanesvitiy) [inline test-e2e/app/Tests/SocialNetworks/Facebook.e2e-spec.ts]: $ is never used
in WEbstorm use "CTRL+ALT+O" as a shortcut for removing unused imports

File: test-e2e/app/Tests/SocialNetworks/Facebook.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    await browser.sleep(3000); // Delay needed to get the post to appear on facebook.
+
+    await switchWindow(0);
+    await browser.get(FacebookPage.baseURL);
+
+    await waitForVisible(FacebookPage.postLinkToDollarStreet);
+    await FacebookPage.postLinkToDollarStreet.click();
+    await switchWindow(1);
+    await browser.waitForAngularEnabled(true);
+    await waitForURLContain('matrix');
```
Comment (ludmilanesvitiy) [inline test-e2e/app/Tests/SocialNetworks/Facebook.e2e-spec.ts]: looks like you copy\paste about 10 lines of code in 'it's - better to move such logic to PO

File: .config/protractor.conf.js (hunk 1 above)
Comment (ludmilanesvitiy) [reply .config/protractor.conf.js]: 1600/900 or 1920/1080

Comment (AntonOlkhovskyi) [reply .config/protractor.conf.js]: Done.

File: test-e2e/app/Tests/SocialNetworks/Facebook.e2e-spec.ts (hunk 3 above)
Comment (AntonOlkhovskyi) [reply test-e2e/app/Tests/SocialNetworks/Facebook.e2e-spec.ts]: Done.

**Review by ludmilanesvitiy — APPROVED** [review]

**Pull request merged**